lib/firespring_dev_commands/templates/aws.rb

```diff
@@ -120,6 +120,61 @@ def create_eol_task!
           end
         end
       end
+
+      # Create the rake task for the hosted zone method
+      def create_dns_logging_activate_task!
+        # Have to set a local variable to be accessible inside of the instance_eval block
+        exclude = @exclude
+
+        DEV_COMMANDS_TOP_LEVEL.instance_eval do
+          namespace :aws do
+            return if exclude.include?(:dns_logging)
+
+            namespace :hosted_zone do
+              namespace :dns_logging do
+                desc 'Activates query logging for all hosted zones by default.' \
+                     'This command should be run from the account the hosted zone(s) reside.' \
+                     "\n\toptionally specify HOSTED_ZONE_GROUP='arn:aws:logs:REGION:ACCOUNT_ID:' to specify the ARN of the target log group." \
+                     "\n\toptionally specify DOMAINS='foo.com,foobar.com' to specify the hosted zones to activate." \
+                     "\n\t\tComma delimited list."
+                task :activate do
+                  route53 = Dev::Aws::Route53.new
+                  route53.hosted_zones(ENV['DOMAINS'].to_s.strip.split(','))
+                  # Use user defined log group. Otherwise, go get the default.
+                  log_group = (ENV['HOSTED_ZONE_GROUP'] || Dev::Aws::Parameter.new.get_value('/Firespring/Internal/Route53/hosted-zone/log-group-arn'))
```

Comment:
I don't think we want a reference to a `/Firespring` reference in the dev commands library?